### DISCUSSION TITLE
Problem with hanging panels

### DISCUSSION THREAD

**Discussion by yanndesa** [Ifsttar/I-Simpa#311] [Q&A]
Hi, I'm trying to do the simulation with the baffles panels in this room. However, SPPS does not recognize their absorption area. Often the results of T15,20,30 end up being even higher than if the room were without the panels. How can I make the panels to be recognized? The mesh is generated correctly
![Sem título2](https://user-images.githubusercontent.com/78579490/155763238-6b18304b-e7b6-404c-8434-901aedd29b5b.png)
![Sem título](https://user-images.githubusercontent.com/78579490/155763243-af5a8373-a510-4a27-8c41-2641e90db068.png)
.

**Comment by sidway** [comment]
Try to make them flat, not with volume.

**Comment by nicolas-f** [comment]
Hi,

You can visualize the propagation of particles using "particle count (render)" parameters in spps. You should see particles bouncing on your panels. I don't know how much I-Simpa can simulate the real effect of hanging panels, may be not very good in low frequency
